src/GovUk.Education.ExploreEducationStatistics.Data.Model/Migrations/20220114144422_E3103UpdateDropAndCreateReleaseToNotDeleteOrphanedFootnotes.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
+using Microsoft.EntityFrameworkCore.Migrations;
+using static GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations.MigrationConstants;
+
+namespace GovUk.Education.ExploreEducationStatistics.Data.Model.Migrations
+{
+    [ExcludeFromCodeCoverage]
+    public partial class E3103UpdateDropAndCreateReleaseToNotDeleteOrphanedFootnotes : Migration
+    {
+        private const string PreviousMigrationId = "20210512112804";
+        private const string MigrationId = "20220114144422";
```

Comment:
See example of very recent stats migrations where we find the `PreviousMigrationId`  in its migration file, make it `public` and then reference it in this class. Reduces the chance of there being any mistake.